src/moj/components/date-picker/_date-picker.scss

```diff
@@ -0,0 +1,278 @@
+.moj-datepicker {
+  position: relative;
+
+  &--fixed-width {
+    .moj-datepicker-input__wrapper {
+      width: 215px;
+    }
+  }
```

Comment:
Rather than fixing the width of the wrapper, we should let people set the input width with the existing GDS classes (e.g. `govuk-input--width-10`)